### PR TITLE
Story3

### PR DESCRIPTION
**Pull request by aidanmaskell**
Created functionality to delete items from database

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -120,4 +120,35 @@ function selectFromDB(PDO $db, string $fieldname) :array {
...
 }
 
+/**
+ * Sets deleted to field to true within db of selected item.
+ *
+ * @param PDO $db the database to be updated
+ * @param string $name the name of the item to be deleted
+ * @return void runs query
+ */
+function deleteFromDB(PDO $db, string $name) {
```
Comment (mporam) [inline functions.php:130]: function doesnt return. the execute function returns a boolean of if the query worked or not, so you can return it and set the return type hint to `bool`

File: functions.php [hunk 2]
```diff
@@ -120,4 +120,35 @@ function selectFromDB(PDO $db, string $fieldname) :array {
...
+
+/**
+ * Returns a message referring to whether there was an error when adding to the database.
+ *
+ * @param bool $add states whether the db has been added to
+ * @param bool $edit states whether the db has been edited
+ * @param bool $del states whether the db has been deleted from
+ * @return string states the outcome message
+ */
+function outcomeMessage(bool $add, bool $edit, bool $del) :string {
```
Comment (mporam) [inline functions.php]: the return type hint should look like:
```php
function outcomeMessage(bool $add, bool $edit, bool $del): string {
```
your space is in the wrong place

File: functions.php [hunk 3]
```diff
@@ -120,4 +120,35 @@ function selectFromDB(PDO $db, string $fieldname) :array {
...
+    if($add || $edit) {
+        return '<h1>Thank you for adding to the database!</h1>';
```
Comment (mporam) [inline functions.php]: where possible its good to only havr 1 return statement, you can do this by putting the string into a variable and then returning that variable at the bottom

File: thankyou.php [hunk 4]
```diff
@@ -4,24 +4,29 @@
```
Comment (mporam) [inline thankyou.php:4]: im not going to ask you to change it, but this file is doing **way** too much. generally a file should have 1 purpose. currently this one has at least 3.

**Review by mporam — CHANGES_REQUESTED** [review]

File: functions.php (hunk 2 above)
Comment (aidanmaskell) [reply functions.php]: I have changed the rest of my functions to match this too

File: functions.php (hunk 3 above)
Comment (aidanmaskell) [reply functions.php]: amended!

File: functions.php (hunk 1 above)
Comment (aidanmaskell) [reply functions.php:130]: amended!

File: thankyou.php (hunk 4 above)
Comment (aidanmaskell) [reply thankyou.php:4]: That sounds wise, I will manage that better in the future

**Review by mporam — APPROVED** [review]

**Pull request merged**